packages/date-time-utilities/src/dateGrid/getDayGrid.ts

```diff
@@ -56,16 +56,14 @@ export const getDayGrid = (options: IDayGridOptions): IDay[][] => {
 
   let selectedDates: Date[] = [];
 
-  if (!!selectedDate) {
-    selectedDates = getDateRangeArray(
-      selectedDate,
-      selectedDateRangeType,
-      firstDayOfWeek,
-      workWeekDays,
-      daysToSelectInDayView,
-    );
-    selectedDates = getBoundedDateRange(selectedDates, minDate, maxDate);
-  }
+  selectedDates = getDateRangeArray(
```

Comment:
Given the [northstar screener failures](https://screener.io/v2/states/H17FwCf88.microsoft-fluentui-fluentui/xgao%2Fremove-unused-deps/1024x768/Chrome/datepickercellexampleshorthandtsx-shows-datepicker-popup-0) I suspect the typings here are wrong (relative to usage), and the `if (selectedDate)` check is actually needed. Maybe best to add back the if() and update the typings to allow undefined?

Reply:
my bad, pushed fix!